docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,49 @@
+# Arktos to Enforce the Multi-tenant with Mizar Network Feature
+
+This document captures the steps applied to an Arktos cluster lab enabling the unique multi-tenant network feature. The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+
+If you would like to try with Flannel cni plugin, please ensure to read [multi-node setup guide](multi-node-dev-cluster.md).
+
+1. Prepare lab machines. Particularly, build arktos-network-controller (as it is not part of arktos-up.sh yet); disable local DNS cache (applicable to 127.0.0.53 name server; as it would cause coreDNS to crash with DNS loopback lookup):
+```bash
+make all WHAT=cmd/arktos-network-controller
```

Comment:
Why not build arktos-network-controller as part of arktos-up if Mizar is CNI? Why does the user need to suffer this extra step to use Mizar?

Reply:
We have updated the code, that will build arktos-network-controller on arktos-up.